cfn-resources/federated-database-instance/cmd/resource/resource.go

```diff
@@ -241,31 +244,46 @@ func handleError(response *http.Response, method string, err error) (handler.Pro
 			Message:          fmt.Sprintf("%s:%s", method, err.Error()),
 			HandlerErrorCode: cloudformation.HandlerErrorCodeAlreadyExists}, nil
 	}
+	if response.StatusCode == http.StatusNotFound {
```

Comment:
is this a new check that was needed before?

Reply:
yes, update operation was not checking if resource to be updated existed before updating, so I added that check before updating and the handling of the 404 status here